tools/runners/run-mongo.js

```diff
@@ -613,10 +632,16 @@ var launchMongo = function (options) {
       }
 
       try {
-        var initiateResult = yieldingMethod(
-          db.admin(), 'command', {replSetInitiate: configuration});
+        yieldingMethod(db.admin(), 'command', {
+          replSetInitiate: configuration,
+        });
       } catch (e) {
-        if (e.message !== 'already initialized') {
+        if (e.message === 'already initialized') {
+          yieldingMethod(db.admin(), 'command', {
+            replSetReconfig: configuration,
+            force: true,
```

Comment:
I believe the use of `force: true` is safe here because this code runs in development, as the Mongo server is starting up, before any Meteor code has a chance to modify the database.

Reference: https://docs.mongodb.com/manual/reference/command/replSetReconfig/